src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ScheduleRuleServiceTest.kt

```diff
@@ -0,0 +1,347 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.activities
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.data.BadDataException
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders.CourseActivityBuilderFactory
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders.CourseScheduleRuleBuilder
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.CourseScheduleRule
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.SlotCategory
+import java.time.LocalDate
+import java.time.LocalTime
+
+class ScheduleRuleServiceTest {
+
+  private val scheduleRuleService = ScheduleRuleService()
+
+  @Nested
+  inner class CreateScheduleRules {
```

Comment:
Corrected the name of the top level class but GitHub is rubbish and thinks it's brand new - but CreateScheduleRules has not been changed at all.